CMakeLists.txt

```diff
@@ -152,8 +152,22 @@ function(configure_kernel kernel)
 endfunction()
 
 message("Configure kernels: ...")
-configure_kernel("/share/jupyter/kernels/xcpp17/")
-configure_kernel("/share/jupyter/kernels/xcpp20/")
+if(NOT EMSCRIPTEN)
+    configure_kernel("/share/jupyter/kernels/xcpp17/")
+    configure_kernel("/share/jupyter/kernels/xcpp20/")
+else()
+    # TODO: Currently jupyterlite-xeus and xeus-lite do not provide
+    # methods to fetch information from the arguments present in the
+    # generated emscripten kernel.
+    # The following needs to be done here :
+    # 1) We need to configure the kernel properly
+    #    Check issue https://github.com/compiler-research/xeus-cpp/issues/185.
+    # 2) Once the above is done we need to add support in jupyterlite-xeus & xeus-lite
+    #    to be able to deal with arguments present in kernel.json
+    # 3) Finally we should fetch the C++ version from the kernel.json file and
+    #    be able to pass it to our wasm interpreter rather than forcing a version.
+    configure_kernel("/share/jupyter/kernels/xcpp20/")
```

Comment:
TODO with respect to supporting more kernels raised here. We would be going ahead with xcpp20 for now but would like to support all.
Please let me know of any questions you'll have or this needs to be rephrased.